packages/spor-theme-react/src/components/tabs.ts

```diff
@@ -0,0 +1,222 @@
+import { tabsAnatomy as parts } from "@chakra-ui/anatomy";
+import type {
+  PartsStyleFunction,
+  PartsStyleInterpolation,
+  PartsStyleObject,
+  StyleFunctionProps,
+  SystemStyleFunction,
+  SystemStyleObject,
+} from "@chakra-ui/theme-tools";
+
+const baseStyleRoot: SystemStyleFunction = (props) => {
+  return {
+    display: "flex",
+    flexDirection: "column",
+  };
+};
+
+const baseStyleTablist: SystemStyleFunction = (props) => {
+  return {
+    display: "flex",
+    alignItems: "center",
```

Comment:
Her kan du bruke [gap](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) for å slippe `_first` og `last` under :) 

```suggestion
    alignItems: "center",
    gap: 0.5,
```